### ISSUE TITLE
Validator misflags png as binary when another binary file exists

### ISSUE THREAD

**Issue by kewisch** [mozilla/amo-validator#411]
https://addons.mozilla.org/en-US/developers/addon/flash-debugger/file/421089/validation

```
Flagged file extensions found.

Warning: Files whose names end with flagged extensions have been found in the add-on.

The extension of these files are flagged because they usually identify binary components. 
Please see http://addons.mozilla.org/developers/docs/policies/reviews#section-binary for 
more information on the binary content review process.

data/flashfirebug.swf
    data/html/images/themes/default/inspector/UIComponents/VolumeBar.png
```

The png file is not a binary file, nor should it be flagged.
